.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: generic
-sudo: false
+sudo: required
```

Comment:
Can you add a comment referencing https://github.com/dask/distributed/pull/1563?

Reply:
Done